### PR TITLE
Replace Redis with Valkey

### LINKED ISSUE #427: Switch Redis to Valkey
## Issue

Redis is switching to a dual license in future versions https://redis.io/blog/redis-adopts-dual-source-available-licensing which may cause licensing issues.

CAPI release currently uses 7.2.5 which does not have this license, however this will eventually go out of support. According to https://redis.io/docs/latest/operate/rs/installing-upgrading/product-lifecycle/#endoflife-schedule August 31, 2025.

## Resolution

We should switch to Valkey instead which is a Redis drop in replacement that is BSD license and part of the Linux Foundation.

It should be a drop in replacement for Redis and we can continue to use the Redis client gem in the cloud controller codebase.

Happy to contribute this.

### PR DESCRIPTION
**Pull request by Samze**
Switch Redis to Valkey

Fixes #427

Tasks:
- [x] Need someone with correct permissions to upload the valkey blob

Once merged:
- [x]  Open a corresponding PR to cf-deployment https://github.com/Samze/cf-deployment/commit/c11377c02bbb9f1a5ea9b32a327affbb289c51dd

On the cloud_controller_ng vm:
```
/var/vcap/packages/valkey/valkey-cli -s /var/vcap/data/valkey/valkey.sock

valkey /var/vcap/data/valkey/valkey.sock> keys *
1) "metrics:cc.requests.outstanding.gauge"
```

* [x] I have viewed signed and have submitted the Contributor License Agreement

* [x] I have made this pull request to the `develop` branch

* [ ] I have run CF Acceptance Tests on bosh lite

### CODE REVIEW

**Comment by philippthun** [comment]
@Samze - I've uploaded the blob using our CI: https://concourse.app-runtime-interfaces.ci.cloudfoundry.org/teams/capi-team/pipelines/capi/jobs/bump-valkey-blob/builds/1

File: config/blobs.yml [hunk 1]
```diff
@@ -94,7 +94,3 @@ postgres/postgresql-11.22.tar.gz:
   size: 26826810
   object_id: d1f8d34c-b438-44e7-7672-5daea8a6da66
   sha: sha256:6445a4e1533c1e8bb616d4a3784bdc4c0226b541f6f0c8d996d9f27d581d49c3
-redis/7.2.5.tar.gz:
```
Comment (Samze) [inline config/blobs.yml:97]: **question** Is there any reason to keep this reference here?

Comment (philippthun) [reply config/blobs.yml:97]: No, when the `packages/redis/*` files are removed, this also needs to be deleted.

**Review by tcdowney — APPROVED** [review]
LGTM!

**Review by a-b — APPROVED** [review]
LGTM

**Review by joaopapereira — APPROVED** [review]
LGTM

**Pull request merged**